.github/workflows/godwoken-tests.yml

```diff
@@ -17,3 +17,9 @@ jobs:
         MANUAL_BUILD_WEB3_INDEXER=true
         WEB3_GIT_URL=https://github.com/${{ github.repository }}
         WEB3_GIT_CHECKOUT=${{ github.ref }}
+        MANUAL_BUILD_POLYJUICE=true
```

Comment:
since godwoken/polyjuice have release new version, is it possible to remove manual build?

Reply:
Will have a try soon.